### ISSUE TITLE
Blazor AOT Follow-ups

### ISSUE THREAD

**Issue by StachuDotNet** [darklang/dark#3832]
The `resource_class` for the `integration-test` step of CI was raised, set to `xlarge`.
We should lower this, especially since that has a `paralellism` of 4.

We should be able to adjust our integration tests to not require a load of Blazor per-test.
Some helpful links:
- https://stackoverflow.com/questions/69572778/playwright-browser-is-reopening-every-test-statement
- https://playwright.dev/docs/test-auth#reuse-signed-in-state
- https://playwright.dev/docs/browser-contexts
- https://stackoverflow.com/questions/68699387/can-you-run-multiple-tests-in-one-browser-context-playwright-javascript

Alternatively, we can make the loading of Blazor much faster, incl. with limited resources, so this isn't an issue.

Once that's done, we can 
- [ ] shorten the playwright timeout
- [ ] lower the resource classes of the integration test CI step
- [ ] remove `await awaitAnalysisLoad(testInfo);` from the `beforeEach` of `integration-tests/tests.ts`

Integration tests still feel a bit flaky. I believe this is _also_ due to the blazor-loading problem

**Comment by StachuDotNet** [comment]
Related: #3833 suggests "move integration tests to a single big machine"

**Comment by StachuDotNet** [comment]
Making good progress here - may roll into #3831 shortly.

**Comment by pbiggar** [comment]
I was recently thinking that the integration tests were fast enough that we'd do better if they were just all run on one (possibly bigger) machine, with more parallelism, instead of paying for 4. Don't know how the calculus has changed after Blazor, but something to consider perhaps.

**Comment by StachuDotNet** [comment]
Good point. This Issue has been effectively merged into #3831 at this point.

Two notes:
- I've implemented the "one big machine" for integration tests within #3831 
- I'm adding a card about loading Blazor only once for tests

**Comment by pbiggar** [comment]
> with more parallelism

I think you go this, but for clarity, I meant to replace using circleci parallelism with using 1 machine and playwright parallelism within it.

**Comment by StachuDotNet** [comment]
Understood. Right now, increasing playwright parallelism seems to cause some flakiness, likely due to demand of ApiServer? I'd like to leave it as-is for that PR for now, but increase paralellism of playwright workers shortly in a separate PR, with the other integration test perf. improvements.

**Comment by pbiggar** [comment]
Yeah, happy to leave for later, let's get this merged.

If it is ApiServer, you might try playing with maxconcurrentconnections: https://github.com/darklang/dark/projects/1#card-79650290
